quick-start-headless-react-css/README.md

```diff
@@ -0,0 +1,10 @@
+# Adaptive Card React with CSS renderer
```

Comment:
Do we want to change the structure so that it something like the following:

React -> json-based-microapp-with-css
React -> json-based-microapp-with-custom-components

Thoughts?

Reply:
agreed - that will clear up some confusion.

Reply:
Updated to: 
- examples/react/json-based-microapp-renderer-with-css
- examples/react/json-based-microapp-renderer-with-custom-components